### PR TITLE
examples: update guppy example with latest fn functionalities

### PR DESCRIPTION
**Pull request by ccirello**
- introduces dependency installation for Ruby functions.

### CODE REVIEW

File: examples/slackbot/guppy/README.md [hunk 1]
```diff
@@ -16,7 +14,9 @@ fn push
 # Create a route to this function on IronFunctions
```
Comment (pedronasser) [inline examples/slackbot/guppy/README.md]: Missing `fn push` somewhere, isn't it?

File: examples/slackbot/guppy/README.md [hunk 2]
```diff
@@ -16,7 +14,9 @@ fn push
...
 fn routes create slackbot /guppy
 # Change the route response header content-type to application/json
-curl -X PUT http://127.0.0.1:8080/v1/apps/slackbot/routes/guppy -d '{ "route": { "headers": { "Content-type": ["application/json"] } } }'
+fn routes headers set slackbot /guppy Content-Type application/json
+# test it remotely
+cat slack.payload | fn call slackbot /guppy2
```
Comment (pedronasser) [inline examples/slackbot/guppy/README.md]: guppy2?

Comment (ccirello) [inline examples/slackbot/guppy/README.md]: Good catch!

File: examples/slackbot/guppy/README.md (hunk 1 above)
Comment (ccirello) [inline examples/slackbot/guppy/README.md]: Actually no. The fn push is right above this line. Check again.

Comment (pedronasser) [inline examples/slackbot/guppy/README.md]: Lol, how I missed it?!? haha

**Pull request merged**